setup.py

```diff
@@ -17,18 +17,15 @@
         'Intended Audience :: Developers',
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
```

Comment:
nit: probably worth keeping `3`, since some automated crawlers use that to determine if a package supports Python 3.